Automattic/wp-calypso
# Update Color Palette to the Most Recent Version

#### Changes proposed in this Pull Request

* Updated the internal color palette to the most recent version.
* Removed 3 fixed color properties in favor of the active color scheme.

#### Testing instructions

This is a purely cosmetic update.

* Make sure the color scheme picker uses the updated thumbnails.

Fixes #36104.
